playbooks/openshift-hosted/private/openshift_default_storage_class.yml

```diff
@@ -4,3 +4,9 @@
   roles:
   - role: openshift_default_storage_class
```

Comment:
Does it make sense to skip running this ``openshift_default_storage_class`` like it is for the `gce` if the role is run again with different values?

Reply:
No, the first run creates default hdd storage class, the second run creates non default ssd storage class.